shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
> The logical conclusion of this (switching from noun "identity" to noun "login") is that we wind up with "Linked logins" and "allow this login to log in to your account?"

Our aim is to use the language that is easiest to understand, not the language which is the most consistent. We don't need to be pedantically consistent when it harms clarity.

> Another reason I shy away from "login" as a noun is that we already use it as a verb.

"Login" is always a noun. "Log in" is a verb. If we use "login" as a verb we should correct it.

If you mean that we shouldn't use both "login" and "log in"... I agree not in the same sentence (because it sounds silly), but I don't think there's a problem with using both of these terms.

> Also, I just asked 2 of the users right now and they both prefer "Identity providers"!

Surprising, but this is a good reason to use "identity provider".

I do want to bring up one other point, though: Our future plans around identity and accounts actually calls for us to disassociate identity from login providers. Will using "identity provider" now create additional confusion later, when we make that change?